src/sentry/static/sentry/app/utils/crashReports.tsx

```diff
@@ -1,30 +1,45 @@
-import {t, tct} from 'app/locale';
+import React from 'react';
 
-type formatConfig = {
-  inProjectSettings?: boolean;
-};
+import {t, tct} from 'app/locale';
 
 export function formatStoreCrashReports(
-  value: number | '',
-  formatConfig: formatConfig = {}
+  value: number | null | '',
+  organizationValue?: number
 ): React.ReactNode {
+  if (value === null && organizationValue) {
```

Comment:
maybe you should check if `organizationValue` is `defined` because as it is of type number it could be 0 and then this condition would be always false.